lib/puppet/pops/lookup/lookup_adapter.rb

```diff
@@ -85,6 +85,7 @@ def lookup_global(key, lookup_invocation, merge_strategy)
       end
     end
   rescue Puppet::DataBinding::LookupError => detail
+    raise detail unless detail.issue_code.nil?
```

Comment:
The reason for this test is to avoid that the exception is re-wrapped and thereby looses it's issue and location. Such re-wrapping will ultimately mean that a new location is assigned to the error. That new location is the file and line of the .pp file where the lookup (or APL) originated.
Ty my knowledge, all errors that have an issue_code originates {{Issues}}. If not, then something is wrong.